Examples/Program.cs

```diff
@@ -172,11 +172,13 @@ static void Main()
                 DownloadSuspendedLiveInterviewData = false,
                 DownloadCapturedMedia = false,
                 DownloadParaData = false,
+                DownloadVarFile = false,
                 DownloadTestInterviewData = true,
                 DownloadFileName = "MyFileName",
                 StartDate = DateTime.Today.ToUniversalTime().ToString("yyyy-MM-ddTHH:mm:ssZ", CultureInfo.InvariantCulture), // UTC time start of today
                 EndDate = DateTime.Today.AddDays(1).ToUniversalTime().ToString("yyyy-MM-ddTHH:mm:ssZ", CultureInfo.InvariantCulture), // UTC time end of today
-                SurveyId = "SomeSurveyId"
+                SurveyId = "SomeSurveyId",
+                SurveyVersion = 637242848690284790 // If the SurveyVersion (Etag) is specified only the surveys matching this version will be downloaded
```

Comment:
Can't you keep the variable name as Etag? Or is that too Techhie? This also saves you the hassle of mapping and such on the api side..

Reply:
We decided internally we will call it Etag but externally Survey version. In the manager it will appear as survey version. See example:

https://xd.adobe.com/view/7095f99c-adc2-4cb7-4878-2de7845ea13f-06a4/